cPP_DSC.adoc

```diff
@@ -1941,6 +1943,14 @@ _If [.underline]#prevent all future authorization attempts indefinitely (i.e., l
 +
 _If [.underline]#factory reset the TOE wiping out all non-permanent SDOs, as described by FDP_FRS_EXT.2# is selected in FIA_AFL_EXT.1.3, the selection-based SFR FDP_FRS_EXT.2 must be claimed._
 
+==== FIA_HMR_EXT.1 Online Authentication Hammering Prevention
+
+FIA_HMR_EXT.1 Online Authentication Hammering Prevention
+
+FIA_HMR_EXT.1.1:: The TSF shall limit the number of authentication attempts for each counter specified in FIA_AFL_EXT.1.1 by [.underline]#[selection: using _[assgnment: integer number greater than or equal to 1,000 iterations]_ of PBKDF as specified in FCS_CKM_EXT.8.1, specifying a maximum of _[assignment: number of attempts]_ that can occur duing _[assignment: time period]_, setting the minimum time period between consecutive attempts to _[assignment: time period]_, preventing future attempts for a time period as defined in FIA_AFL_EXT.1.3, _[assignment: other anti-hammering method]_]#.
```

Comment:
Once again, this will not work with Apple: they have a much smaller round count of PBKDF but "back it up" with another cryptographic operation that is a similar iteration operation as PBKDF.

Reply:
So I wrote this intentionally this way, but left an open assignment at the end. So basically if someone just used PBKDF, they should have a high iteration count, or they can do other things related to time periods, etc. At the end though, if they are doing something not explicitly mentioned here, they can describe that in the open assignment.

My understanding of the way that Apple works (admittedly not very specific) is that they could possibly meet the second selection (basically specifying they are limiting the number of attempts that can be performed over time, with the method for doing so being their other operation), but they could definitely describe it in the open assignment.

This could be further refined but I don't want to make something specific for Apple, but to make sure that Apple can either describe what they are doing for the claim or have something generic enough they could use as their selection and then provide their explanation to meet it.